website/blog/2023-08-11-wavemaker10x-end-of-support.md

```diff
@@ -1,25 +1,27 @@
 ---
-title: "Transitioning from WaveMaker 10 to WaveMaker 11"
-author: Swetha Kundaram
+title: "Ending Support to WaveMaker 10 Projects"
+author: Ritupurna Lenka
 ---
 ---
 
 WaveMaker strives to provide the best possible experience to all our users by regularly enhancing and upgrading our products. With this in mind, we are excited to announce the upcoming release of WaveMaker Version 11.4, which will introduce various new features and improvements, such as React Native features, tightly-coupled team collaboration, and reimport of backend APIs, and more exciting features. 
```

Comment:
In this line we should include that when we released WM 11 back in (insert date) we enabled customer to remain on WM10 and plan their migration. This flexibility has been available for past few months and with 11.4 release this will not be offered.